### DISCUSSION TITLE
please help (WARNING:semantic_router.utils.logger:Retrying in 4 seconds... INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings "HTTP/1.1 429 Too Many Requests" INFO:openai._base_client:Retrying request to /embeddings in 0.783549 seconds)

### DISCUSSION THREAD

**Discussion by leotrax3d** [judahpaul16/gpt-home#68] [Q&A] [answered]
![Screenshot 2024-12-16 145930](https://github.com/user-attachments/assets/f91e991b-c29b-4de1-9fa6-71449a5eaaec)i 

when i start the system i get the following error message. i need quick and accurate help. thanks in advance

**Comment by judahpaul16** [comment, marked as answer]
First make sure you have at least a dollar in your [platform.openai.com](https://platform.openai.com/settings/organization/billing/overview) account. Check your `~/.bashrc` (don't post  it) but verify your OPENAI_API_KEY variable is set. If it is try recreating the container.
```bash
source ~/.bashrc
docker stop gpt-home
docker system prune -af
curl -s https://raw.githubusercontent.com/judahpaul16/gpt-home/main/contrib/setup.sh | bash -s -- --no-build
```

**Reply by leotrax3d** [reply]
Thanks

I solve it. The 1dollar was Missing

**Reply by judahpaul16** [reply]
No problem.
